public/sass/pages/_login.scss

```diff
@@ -162,18 +162,17 @@ select:-webkit-autofill:focus {
   transform: tranlate(0px, 0px);
   transition: 0.25s ease;
 
-  &.add {
-    transform: translate(0px, -320px);
-    &.hidden {
-      display: none;
-    }
+  &.hidden {
```

Comment:
Do you feel like migrating _login.scss to Emotion? Or this may be worth another PR

Reply:
Yeah, I could do that after this is merged, just to have some closure I guess :smiley: